plugins/source/aws/resources/services/auditmanager/assesments.go

```diff
@@ -0,0 +1,56 @@
+package auditmanager
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/auditmanager"
+	"github.com/aws/aws-sdk-go-v2/service/auditmanager/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func Assessments() *schema.Table {
+	tableName := "aws_auditmanager_assessments"
+	return &schema.Table{
+		Name:                tableName,
+		Description:         `https://docs.aws.amazon.com/audit-manager/latest/APIReference/API_Assessment.html`,
+		Resolver:            fetchAssessments,
+		PreResourceResolver: getAssessment,
+		Multiplex:           client.ServiceAccountRegionMultiplexer(tableName, "appsync"),
```

Comment:
Is `appsync` the correct multiplexer to be using here? Just surprising that it's not `auditmanager` or something related to that

Reply:
It was wrong, I have already updated it in https://github.com/cloudquery/cloudquery/pull/12573/commits/a5e67e0097a8383d85a3eb51cab9487890be4326